WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,8 +1,5 @@
-- Theme/default/custom color palettes
-- Email app choice when verifying email at signup/login
-- No nested comments if parent comment is deleted
-- Images adjust after new mid-capture orientation
-- Progress bar fix for My Site icon edits
-- Reorganized default Insights list
-- Keyboard won’t cover login error message
-- Logout button only has to be pressed once
+* [*] Notification: Adds Weekly Roundup switch to App info -> Notification menu to disable them for all sites. [https://github.com/wordpress-mobile/WordPress-Android/pull/16118]
+* [*] Block Editor: Add GIF badge for animated GIFs uploaded to Image blocks [https://github.com/WordPress/gutenberg/pull/38996]
+* [*] Block Editor: Small refinement to media upload errors, including centering and tweaking copy. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4597]
+* [*] Block Editor: Fix issue with list's starting index and the order [https://github.com/WordPress/gutenberg/pull/39354]
```

Comment:
All these equally apply to WP and JP, so no manual amending to be done for this sprint! (cc @leandroalonso to confirm, just as a matter of principle 😛 )